### ISSUE TITLE
No license

### ISSUE THREAD

**Issue by Azrael-DC** [as3/as3-utils#16]
Have you guys considered how you are going to license this project? GPL 2 or 3? MIT? Apache? 

Inquiring minds want to know.

**Comment by chrisdeely** [comment]
Good question, I don't think anyone has really given it thought, as this isn't a cohesive library - just a collection of helpful utility methods.

I lean towards MIT, does anyone else have suggestions?

**Comment by mimshwright** [comment]
I usually use MIT. I think the general intent was for this to be totally open and used for whatever anybody wants to use it for. If nobody else objects in the next day or two, i'll make it so.

**Comment by chrisdeely** [comment]
Totally agree. If you have a copy of the MIT license handy please throw it up there.  Thanks

**Comment by mimshwright** [comment]
I was about to post the MIT license to our readme file then I remembered that this code is gathered from all over the place. Some of it might actually be licensed already. So I'm not sure it's safe for us to say that it's all MIT unless we include all the individual licenses. For this license to mean anything, I think we have to go through all the code we took from other sources.

That being said, the INTENT of the library is very clear. It's free for everyone to use and modify and we will never charge for it. All the code included in the library was taken from open source projects or written specifically for this project (as far as I know).
